src/modules/qt/gps_parser.cpp

```diff
@@ -937,6 +940,22 @@ void process_gps_smoothing(gps_private_data gdata, char do_processing)
                 nr_atemp++;
             }
 
+            //power
+            if (gp_r[i].power != GPS_UNINIT) {
+                if (power != GPS_UNINIT && nr_power > 0 && nr_power <= nr_one_minute * 60) {
```

Comment:
> nr_one_minute * 60

Not very important but I recommend removing the "* 60" part and just leave the one minute default allowed gap. While I see that Strava fills in with 0 during pauses, I can imagine some other service not bothering to write anything and you'd have a period of up to 1 hour of not pedaling where the power would be interpolated for no reason.